scripts/assets/maven_dependencies.textproto

```diff
@@ -0,0 +1,875 @@
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "com.squareup.okio:okio:2.2.2"
+  artifact_version: "2.2.2"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "com.squareup.retrofit2:converter-moshi:2.5.0"
+  artifact_version: "2.5.0"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "com.squareup.retrofit2:retrofit:2.9.0"
+  artifact_version: "2.9.0"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "com.squareup:kotlinpoet:1.6.0"
+  artifact_version: "1.6.0"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "de.hdodenhof:circleimageview:3.0.1"
+  artifact_version: "3.0.1"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "io.fabric.sdk.android:fabric:1.4.7"
+  artifact_version: "1.4.7"
+  license {
+    license_name: "Terms of Service for Firebase Services"
+    original_link: "https://fabric.io/terms"
+    direct_link_only {
+      url: "https://firebase.google.com/terms"
+    }
+    is_original_link_invalid: true
+  }
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://raw.githubusercontent.com/firebase/firebase-android-sdk/master/LICENSE"
+    scrapable_link {
+      url: "https://raw.githubusercontent.com/firebase/firebase-android-sdk/master/LICENSE"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "io.github.chaosleung:pinview:1.4.4"
+  artifact_version: "1.4.4"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
+    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    scrapable_link {
+      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }
+  }
+}
+maven_dependency {
+  artifact_name: "javax.annotation:javax.annotation-api:1.3.2"
+  artifact_version: "1.3.2"
+  license {
+    license_name: "CDDL + GPLv2 with classpath exception"
+    original_link: "https://github.com/javaee/javax.annotation/blob/master/LICENSE"
```

Comment:
Why not point to a static blob reference to this license, too, rather than pulling it into the licenses repo?

Reply:
Done.